app/views/trades/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= stylesheet_link_tag 'form' %>
+
+<div class="heading">
+  <div class="left-link">
+    <%= link_to raw('<i class="fas fa-arrow-left"></i>'), user_group_path(current_user, @group), class: "back-icon" %>
+  </div>
+  <h1 class="center-heading">Create Transaction</h1>
+</div>
+
+<div class="forms">
+  <%= form_for @trade, url: user_group_trades_path(current_user, @group) do |f| %>
+
+    <div class="field">
+      <%= f.text_field :name, placeholder: "Enter transaction name" %>
+    </div>
+    <div class="field">
+      <%= f.number_field :amount, placeholder: "Enter transaction amount" %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit 'Create' %>
```

Comment:
- [ OPTIONAL ] Amazing job working on the project overall but I think it would be great if you add an input in the transaction form since it is one of the requirements and its importance shows when the user adds a new transaction but belongs to the another category so it should be in the group that related to that category ⬇💯🔥🎉

![image](https://github.com/muneeb3123/Budget-App/assets/58553711/6fe9ec1f-d43e-4afb-bdfe-c9dec8dfbae4)

Reply:
@Reem-lab   i can do it